### PR TITLE
Mutefix

### PR DESCRIPTION
**Pull request by kswanson33**
Fixes the Mute! extension for the new dashboard. Since post types are obsolete, users will no longer be able to mute based on post type. Instead, users have the option to mute original posts, reblogged posts, or both.

### CODE REVIEW

File: Extensions/mute.js [hunk 1]
```diff
@@ -566,53 +563,54 @@ XKit.extensions.mute = new Object({
...
+		// 	if (user_object.audio === true) { m_audio_class = "selected"; }
+		// 	if (user_object.video === true) { m_video_class = "selected"; }
+		// 	if (user_object.asks === true) { m_asks_class = "selected"; }
+		// 	if (typeof user_object.reblogs !== "undefined") {
+		// 		if (user_object.reblogs === true) { m_reblogs_class = "selected"; }
+		// 	}
+		// 	if (typeof user_object.originals !== "undefined") {
+		// 		if (user_object.originals === true) { m_originals_class = "selected"; }
+		// 	}
+		// }
```
Comment (marcustyphoon) [inline Extensions/mute.js:594]: It looks like the above section might still be needed in order to update the option checkboxes to the current settings!

Comment (kswanson33) [reply Extensions/mute.js]: Thanks! Good eye; should be fixed now.

**Comment by marcustyphoon** [comment]
- Make sure to bump the version number up at the very top!
- Using `display: none` like the old version of the extension does makes Tumblr's j/k shortcut keys stop working on the new dash. You can use Becca's `XKit.interface.hide` function like this: https://github.com/new-xkit/XKit/pull/1983/commits/7735b7f5d3dc7bb33ca6d01132603ea327d4d7d9
- This isn't related to any changes you made, but the `destroy()` function used when you disable the extension appears to have been broken for 6+ years. This guy could use a corresponding `remove`: https://github.com/new-xkit/XKit/blob/811327ad484d75efc24c270c0be5086dd97aff0c/Extensions/mute.js#L117 and this section is copied into `destroy()`, except it's missing its final line there: https://github.com/new-xkit/XKit/blob/811327ad484d75efc24c270c0be5086dd97aff0c/Extensions/mute.js#L375-L379

Otherwise, looks good to me!

**Comment by kswanson33** [comment]
>     * Make sure to bump the version number up at the very top!

@marcustyphoon I have to confess I don't know much about updating versions ... since the current version is 2.4.0, would this be 2.4.1, or would it go up to 2.5.0?

File: Extensions/mute.js [hunk 2]
```diff
@@ -329,46 +330,46 @@ XKit.extensions.mute = new Object({
 				continue;
 			}
 
-			if (type === "note" && XKit.extensions.mute.muted[i].asks === true) {
-				return true;
-			}
+			// if (type === "note" && XKit.extensions.mute.muted[i].asks === true) {
```
Comment (hobinjk) [inline Extensions/mute.js:333]: I'd prefer deleting code to commenting it out since we have the git history

**Review by hobinjk — CHANGES_REQUESTED** [review]
Tested locally and functionality LGTM. Thank you for the contribution!

One thing I would like to see changed is deleting code instead of commenting out since a fair fraction of this file ends up being out-of-date commented-out code. That being said, it's easy enough for me to do this if you don't have the time so let me know if you want me to make the final changes.

**Comment by kswanson33** [comment]
@hobinjk Took out (my) commented code blocks. Thanks for reviewing and for letting me know about the best practice there. Good to tidy up!

**Pull request closed without merging**